contrib/win32/openssh/OpenSSHUtils.psd1

```diff
@@ -1,44 +1,21 @@
-#
-# Module manifest for module 'OpenSSHUtils'
-#
-# Generated on: 6/9/2017
-#
-
 @{
-
-# Script module or binary module file associated with this manifest
-ModuleToProcess = 'OpenSSHUtils.psm1'
-
-# Version number of this module.
-ModuleVersion = '1.0.0.1'
-
-# ID used to uniquely identify this module
-GUID = '08285dee-3d08-476b-8948-1a7e2562c079'
-
-# Author of this module
-Author = 'Yanbing Wang'
-
-# Company or vendor of this module
-CompanyName = ''
-
-# Copyright statement for this module
-Copyright = ''
-
-# Description of the functionality provided by this module
-Description = 'Configure OpenSSH for Windows related security settings like file owner and permissions.'
-
-# Functions to export from this module
-FunctionsToExport = '*'
-
-# Cmdlets to export from this module
-CmdletsToExport = '*'
-
-# Variables to export from this module
-VariablesToExport = '*'
-
-# Aliases to export from this module
-AliasesToExport = '*'
-
-# Minimum version of the Windows PowerShell engine required by this module
-PowerShellVersion = '2.0'
+    ModuleToProcess = 'OpenSSHUtils.psm1'
+    ModuleVersion = '0.0.2.0'
+    GUID = '08285dee-3d08-476b-8948-1a7e2562c079'
+    Author = 'Microsoft Corporation'
+    CompanyName = 'Microsoft Corporation'
+    Copyright = '© Microsoft Corporation. All rights reserved.'
+    Description = 'Utilities and functions for configuring OpenSSH on Windows.'
+    FunctionsToExport = 'Repair-SshdConfigPermission', 'Repair-SshdHostKeyPermission', 'Repair-AuthorizedKeyPermission', 'Repair-UserKeyPermission', 'Repair-UserSshConfigPermission' 
```

Comment:
There is customers requests that pushing Repair-FilePermission. If we don't export it, it will break them.
Enable-Privilege, Get-UserAccount, Get-UserSID are called my our pester tests.  if we don't export them, we need to update the tests.

Reply:
Then we should call it something else...looking into exactly what it does so we can come up with something better.

Also, not sure how we address (what should be) internal functions getting hit in e2e tests (finally got a working build on a machine so I don't have to keep submitting commits). Let me bug some people in PowerShell and see how they're dealing with that sort of thing.